.omeroci/test-data

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+set -eux
+
+OMERO=${OMERO_DIST:-/opt/omero/server/OMERO.server}/bin/omero
+$OMERO login -C -s localhost -u root -w omero
+$OMERO group add "بيئة مجهرية مفتوحة"
+$OMERO user add -P ome микроскопом-пони микроскопом пони "بيئة مجهرية مفتوحة"
+# Login as микроскопом-пони fails with a unicode error so create
```

Comment:
Make sure this is recorded somewhere.

Reply:
https://trello.com/c/p6cTWwZG/511-cli-login-unicode-username-fails